ext/rugged/rugged_rebase.c

```diff
@@ -67,6 +69,63 @@ VALUE rugged_rebase_new(VALUE klass, VALUE owner, git_rebase *rebase)
 	return rb_rebase;
 }
 
+struct get_annotated_commit_args {
```

Comment:
@carlosmn Do you think we could convert this pattern (lines 72 up to 127) into a macro?